stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/MessageListView.kt

```diff
@@ -545,6 +546,35 @@ public class MessageListView : ConstraintLayout {
         super.onDetachedFromWindow()
     }
 
+    /**
+     * Add a listener that will be notified of any changes in scroll state or position. The listener
+     * is added to the inner [RecyclerView] that is used to display a list of message list items.
+     *
+     * @param listener The listener to register.
+     */
+    public fun addOnScrollListener(listener: RecyclerView.OnScrollListener) {
+        binding.chatMessagesRV.addOnScrollListener(listener)
+    }
+
+    /**
+     * Removes a listener that was notified of any changes in scroll state or position. The listener
+     * is removed from the inner [RecyclerView] that is used to display a list of message list items.
+     *
+     * @param listener The listener to unregister.
+     */
+    public fun removeOnScrollListener(listener: RecyclerView.OnScrollListener) {
+        binding.chatMessagesRV.removeOnScrollListener(listener)
+    }
+
+    /**
+     * Returns the inner [RecyclerView] that is used to display a list of message list items.
+     *
+     * @return The inner [RecyclerView] with messages.
+     */
+    public fun getInnerMessageList(): RecyclerView {
+        return binding.chatMessagesRV
+    }
```

Comment:
What's the use case for exposing this?

Reply:
Decided to expose just the inner RecyclerView.